Java/FizzBuzz_ramgsuri.java

```diff
@@ -0,0 +1,14 @@
+import java.util.stream.IntStream;
+
+public class FizzBuzz_ramgsuri {
+
+    public static void main(String args[]){
+
+        IntStream.rangeClosed(0, 100).mapToObj(
+                i -> i % 3 == 0 ?
+                        (i % 5 == 0 ? "FizzBuzz" : "Fizz") :
+                        (i % 5 == 0 ? "Buzz" : i))
+                .forEach(System.out::println);
+    }
+
+}
```

Comment:
Hi, @ramgsuri This same approach is already present in https://github.com/ashish-r/Fizz-Buzz/blob/main/Java/FizzBuzzv2.java You can modify the file and fix that solution.

Reply:
Hi @ashish-r I would suggest you not to merge incorrect solutions and then fixing later instead accept correct solutions.